antiope-sqs/src/Antiope/SQS.hs

```diff
@@ -139,6 +142,39 @@ forAllMessages' :: (MonadUnliftIO m, HasEnv env)
 forAllMessages' env recMsg mode process =
   forAllMessages'' env recMsg mode (const process)
 
+-- | Reads all messages from the queue into a LAZY list.
+-- This function will read the provided SQS queue one batch at a time as
+-- the list consumption is going.
```

Comment:
Should read "as the list is consumed".

It should also document what happens when the queue is empty.  Does it block or does the list terminate?

Reply:
What happens if empty is controlled by `ConsumerMode` parameter, as in other (existing) functions.
When it is `Forever`, it'll loop over the queue and produce an infinite list.
When it is `Drain`, it'll stop when the queue is empty.